tools/system_message.toit

```diff
@@ -34,74 +34,35 @@ handle_system_message encoded_system_message snapshot_content:
   else:
     pipe.print_to_stdout m.stringify
 
-usage prog_name:
-  pipe.print_to_stderr """
-    Usage:
-      $prog_name <snapshot>
-      $prog_name <snapshot> <system message or heap_dump file>
-      $prog_name <snapshot> -b <base64-encoded-ubjson>
-      $prog_name [--snapshot|-s <snapshot>] --message|-m <base64-encoded-ubjson>
-      # Eg snapshot file can be toit.run.snapshot
-
-    If no system-message file is given, the stack trace is read from stdin."""
-  exit 1
-
 main args:
   prog_name := "system_message"
-  if args.size < 1: usage prog_name
-  legacy_args := (args[0].ends_with ".snapshot") and (not args[0].starts_with "-")
+
+  // Use arguments library.
+  parsed := null
+  command := cli.Command prog_name
```

Comment:
The command name for the top-level command isn't used. Maybe use "root" instead like in other places?